Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -45,6 +45,7 @@
 #include "../Hoi4Outputter/ScriptedTriggers/OutScriptedTriggers.h"
 #include "../Hoi4Outputter/States/HoI4StatesOutputter.h"
 #include <fstream>
+#include "../Hoi4Outputter/outDifficultySettings.cpp"
```

Comment:
You need to create a .h file for this function and include it instead of the .cpp file.